cudapoa/src/main.cpp

```diff
@@ -0,0 +1,276 @@
+/*
+* Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+*
+* NVIDIA CORPORATION and its licensors retain all intellectual property
+* and proprietary rights in and to this software, related documentation
+* and any modifications thereto.  Any use, reproduction, disclosure or
+* distribution of this software and related documentation without an express
+* license agreement from NVIDIA CORPORATION is strictly prohibited.
+*/
+
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <claraparabricks/genomeworks/cudapoa/utils.hpp> // for get_multi_batch_sizes()
+#include "application_parameters.hpp"
+
+namespace claraparabricks
+{
+
+namespace genomeworks
+{
+
+namespace cudapoa
+{
+
+std::unique_ptr<Batch> initialize_batch(int32_t mismatch_score, int32_t gap_score, int32_t match_score, bool msa, bool banded_alignment, const double gpu_mem_allocation, const BatchSize& batch_size)
```

Comment:
let's make this multi line

Reply:
I was expecting the autoformatter to do this :sweat_smile: Let me go fix this.

Reply:
I will look into why the formatter didn't take care of this...